sidebars.js

```diff
@@ -129,6 +129,11 @@ MAFIA_Build: [
       id: 'build_script/HW_elab_simulate',
       label: 'HW elaboration & simulation',
     },
+    {
+      type: 'doc',
+      id: 'build_script/advanced build options',
```

Comment:
Should not have spaces in the name of a file or folder.
Please review and fix

Reply:
fixed